source/clear-linux/network/network-bonding.rst

```diff
@@ -0,0 +1,131 @@
+.. _network-bonding:
+
+Network Bonding
+###############
+
+Network bonding is a technique for combining multiple network interfaces into
+a single, logical interface, providing some combination of redundancy and
+bandwidth aggregation.
+
+|CLOSIA| includes the bonding_ and team_ drivers. Here, we'll provide an
+example of configuring systemd to use the ``bonding`` driver.
+
+First, if it does not already exist, as root, create the
+``/etc/systemd/network`` directory:
```

Comment:
Wouldn't it be simpler to say, "... create the directory below:" as that would avoid the duplication of that string (which makes maintenance of this doc easier)?